content/pages/burials-and-memorials/burial-planning.md

```diff
@@ -27,6 +27,7 @@ Yes. You can plan ahead to make the process of applying for a burial in a VA nat
 - Send it by U.S. mail to: 
   
 <dl class="va-address-block">
+    <dt></dt>
```

Comment:
It doesn't seem like a definition list makes sense for an address, does it? Should we switch this whole thing to be in a `p` tag?